eng/imports/RepoLayout.props

```diff
@@ -8,7 +8,6 @@
     <ArtifactsConfigurationDir>$(ArtifactsDir)$(Configuration)\</ArtifactsConfigurationDir>
     <ArtifactsBinDir>$(ArtifactsConfigurationDir)bin\</ArtifactsBinDir>
     <ArtifactsObjDir>$(ArtifactsConfigurationDir)obj\</ArtifactsObjDir>
-    <ArtifactsLogDir>$(ArtifactsConfigurationDir)log\</ArtifactsLogDir>
```

Comment:
Other than the directory being created, this property was not used by our build.